l10n_ch_qr_bill_scan/readme/INSTALL.rst

```diff
@@ -0,0 +1,11 @@
+This module depends on `account_invoice_import` which is located in `edi` and has dependencies on `community-data-files`.
```

Comment:
Then they should be included in the `oca_dependencies.txt` see https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt

Reply:
Hello @TDu `oca_dependencies.txt` was for branches that relied on Travis for the ci; it is not necessary anymore now that dependencies are atomically installed by pip; I will open a PR to improve the document you mention

Reply:
Ok I don't know that. Here it works because we have the `test_requirements.txt` that pull the dependencies but will it still work when that commit is removed ?

Reply:
- `test_requirements.txt` will be removed from this PR once the needed PRs will have been merged
- at that moment, modules will have been made available directly from the [OCA's wheelhouse](https://wheelhouse.odoo-community.org/oca-simple/), so tests will continue to run properly

Reply:
Thanks for the explanation, I didn't know that.